nat-lab/bin/inject-icmp-host-unreachable

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+
+import argparse
+from scapy.all import *
+
+NEXTHOP = 1300
+
+
+def callback(pkt):
+    if not pkt.haslayer(ICMP):
+        return
+
+    if pkt[ICMP].type == 8:
```

Comment:
A super small thing, but maybe it is worth adding some sort of constant or comment explaining, that `8` is the `Echo Request` type?

Reply:
Done